usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -44,6 +44,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/program_activity/count/](/api/v2/agency/012/program_activity/count/)|GET| Returns the count of Program Activity categories for the agency in a single fiscal year |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/sub_agency/](/api/v2/agency/012/sub_agency/)|GET| Returns a list of sub-agencies and offices with obligated amounts, transaction counts and new award counts for the agency in a single fiscal year |
 |[/api/v2/agency/<TOPTIER_AGENCY_CODE\>/sub_agency/count/](/api/v2/agency/012/sub_agency/count/)|GET| Returns the number of sub-agencies and offices for the agency in a single fiscal year |
+|[/api/v2/agency/<TOPTIER_AGENCY_CODE\>//api/v2/agency/<TOPTIER_AGENCY_CODE\>/subcomponents/<BUREAU_SLUG\>/](/api/v2/agency/012/subcomponents/farm-service-agency/)|GET| Returns a list of federal_accounts by bureau for the agency in a single fiscal year |
```

Comment:
Similar question here as with the other PR regarding `subcomponents` vs `sub_components`.